src/responder/multiResponder.ts

```diff
@@ -33,16 +34,21 @@ export class MultiResponder {
      *   The signer used to sign transaction created by this responder. This responder
      *   requires exclusive use of this signer.
      * @param gasEstimator
+     * @param responderBalanceTreshold
+     *   This value respresents the minimum treshold the responder balance can reach before a
+     *   "low balance warning will be issued"
      */
     public constructor(
         public readonly signer: ethers.Signer,
         public readonly gasEstimator: GasPriceEstimator,
         private readonly chainId: number,
         store: ResponderStore,
-        public readonly address: string
+        public readonly address: string,
+        public readonly responderBalanceTreshold: number
     ) {
         this.broadcast = this.broadcast.bind(this);
         this.zStore = store;
+        this.balanceTreshold = responderBalanceTreshold;
```

Comment:
You dont need to set this here, since responderBalanceThreshold is declared `public readonly` in the constructor. https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/

I think the name `balanceThreshold` is good for this property.